### PR TITLE
[Word] (troubleshooting) Add guide

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Remove article dedicated to null values and add info to newly-created troubleshooting guide

### CODE REVIEW

File: docs/word/word-add-ins-troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: Troubleshooting Word add-ins
```
Comment (lindalu-MSFT) [inline docs/word/word-add-ins-troubleshooting.md:2]:
```suggestion
title: Troubleshoot Word add-ins
```

File: docs/word/word-add-ins-troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+description: Learn how to troubleshoot development errors in Word add-ins.
+ms.date: 02/01/2023
+ms.localizationpriority: medium
+---
+
+# Troubleshooting Word add-ins
```
Comment (lindalu-MSFT) [inline docs/word/word-add-ins-troubleshooting.md:8]:
```suggestion
# Troubleshoot Word add-ins
```

Comment (lindalu-MSFT) [reply docs/word/word-add-ins-troubleshooting.md:8]: Considering what people might enter into search, there is a strong argument to leave the H1 as is. Up to you!

File: docs/word/word-add-ins-troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+This article discusses troubleshooting issues that are unique to Word. Use the feedback tool at the end of the page to suggest other issues that can be added to the article.
+
+## Layout breaks when using `insertHtml` while cursor is in content control in header
+
+This issue may occur in the following case.
```
Comment (lindalu-MSFT) [inline docs/word/word-add-ins-troubleshooting.md:14]:
```suggestion
This issue may occur in the following cases.
```
Do all three need to be present in order for this to occur or just one?

**Review by lindalu-MSFT — APPROVED** [review]

File: docs/word/word-add-ins-troubleshooting.md (hunk 3 above)
Comment (ElizabethSamuel-MSFT) [reply docs/word/word-add-ins-troubleshooting.md:14]: all three are needed for the scenario - maybe I can say "conditions" instead?

Comment (lindalu-MSFT) [reply docs/word/word-add-ins-troubleshooting.md]: "This issue may occur when the following conditions are met."

**Pull request merged**